src/contracts/pods/EigenPodStorage.sol

```diff
@@ -71,10 +71,16 @@ abstract contract EigenPodStorage is IEigenPod {
     /// @notice The current checkpoint, if there is one active
     Checkpoint internal _currentCheckpoint;
 
+    /// @notice An address with permissions to call `startCheckpoint` and `verifyCheckpointProofs`, set
+    /// by the podOwner. This role exists to allow a podOwner to designate a hot wallet that can call
+    /// these methods, allowing the podOwner to remain a cold wallet that is only used to manage funds.
+    /// @dev If this address is NOT set, only the podOwner can call `startCheckpoint` and `verifyCheckpointProofs`
```

Comment:
typo here, should be `verifyWithdrawalCredentials`

Reply:
[b23f172](https://github.com/Layr-Labs/eigenlayer-contracts/pull/629/commits/b23f1720212c84c44874c5247baa59bdf3152770)